manim/mobject/types/vectorized_mobject.py

```diff
@@ -414,7 +433,57 @@ def set_sheen_direction(self, direction, family=True):
             self.sheen_direction = direction
         return self
 
-    def set_sheen(self, factor, direction=None, family=True):
+    def rotate_sheen_direction(self, angle: np.ndarray, axis: float = OUT, family=True):
+        """Rotates the direction of the applied sheen.
+
+        Parameters
+        ----------
+        angle : :class:`float`
+            Angle by which the direction of sheen is rotated.
+        axis : :class:`numpy.ndarray`
+            Axis of rotation.
+
+        Examples
+        --------
+        Normal usage::
+
+            Circle().set_sheen_direction(UP).rotate_sheen_direction(PI)
+
+        See Also
+        --------
+        :meth:`~.VMobject.set_sheen_direction`
+        """
+        if family:
+            for submob in self.get_family():
+                submob.sheen_direction = rotate_vector(
+                    submob.sheen_direction, angle, axis
+                )
+        else:
+            self.sheen_direction = rotate_vector(self.sheen_direction, angle, axis)
+        return self
+
+    def set_sheen(self, factor, direction: np.ndarray = None, family=True):
+        """Applies a color gradient from a direction.
+
+        Parameters
+        ----------
+        factor : :class:`float`
+            The extent of lustre/gradient to apply. If negative, the gradient
+            starts from black, if positive the gradient starts from white and
+            changes to the current color.
+        direction : :class:`numpy.ndarray`, optional
+            Direction from where the gradient is applied.
+
+        Examples
+        --------
+        .. manim:: SetSheen
+            :save_last_frame:
```

Comment:
The example currently doesn't render correctly due to a missing newline before the code block.

```suggestion
            :save_last_frame:

```